changelog/fix_an_error_for_safe_navigation_when_negation_and_safe_navigation.md

```diff
@@ -0,0 +1 @@
+* [#10946](https://github.com/rubocop/rubocop/pull/10946): Fix `Style/SafeNavigation` for negated method call with a safe navigation. ([@fatkodima][])
```

Comment:
Thanks for the catchup and fixing!
This changelog entry is unneeded because the patch to changes that have not yet been released.

Reply:
Ah, sure 🤦